Introduction/hello.java

```diff
@@ -1,5 +1,5 @@
 public class hello{
     public static void main(String[] args) {
-        System.out.print("Hello World"+"\n"+"I am A java beginner\n");
+        System.out.printf("Hello World"+"%n"+"I am A java beginner%n");
```

Comment:
there's an error here use ``` println ``` instead of  ``` printf ```

Reply:
%n is a format to print os dependent next line instead of writing fixed according to the owner os.

Reply:
if ln is used it will print %n as a character rather than next line according to os

Reply:
Thanks for the insight

Reply:
Did you synch your branch before I can merge your Pr?

Reply:
yes, you can merge the PR